tests/update/updates_packagekit_gpk.pm

```diff
@@ -55,6 +63,10 @@ sub run {
 
         if (match_has_tag("updates_restart_application")) {
             assert_and_click("updates_restart_application");
+
+            # We also need the daemon to reload to pick up libzypp updates
+            # Force reloading of packagekitd (bsc#1075260, poo#30085)
+            tell_packagekit_to_quit;
```

Comment:
Actually, we already have a utility function called `pkcon_quit` for that :-) Please use this one.

Reply:
`pkcon_quit` masks the service and cannot be used, as subsequent calls to `pkcon` will fail.